packages/govern-console/README.md

```diff
@@ -0,0 +1,68 @@
+This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
```

Comment:
let's remove boilerplate and ideally do something quick and neat

Reply:
Sure :)